testdata/build.toml

```diff
@@ -1,6 +1,6 @@
 repo = "https://github.com/project-oak/oak"
 commit_hash = "0f2189703c57845e09d8ab89164a4041c0af0a62"
 builder_image = "gcr.io/oak-ci/oak@sha256:53ca44b5889e2265c3ae9e542d7097b7de12ea4c6a33785da8478c7333b9a320"
-command = "./scripts/runner build-functions-server"
+command = ["./scripts/runner", "build-functions-server"]
```

Comment:
This seems to reverse the change done in https://github.com/project-oak/transparent-release/pull/9. Are we sure we want to do that?

Reply:
Yes. I had to revert the change in #9 to make the `verify` functionality work. 

It is important to pass an array in `args = append(args, b.Command...)` instead of a string, otherwise the command execution in the following line fails: 
https://github.com/project-oak/transparent-release/blob/b5c32a38184c44821e2d5abe71022e5fcf2b2ba5/common/common.go#L133

I could keep the input command as a single string, and split that into an array and use the array for building `args` right before executing the command. But I am not sure how that addresses the concern that @tiziano88 raised on #9. 

So let's get back to #9 after we have this test in CI :)

Reply:
But feel free to create a PR to fix the array part, keeping the command as a single string. Then I'll rebase this on your change, instead of reverting #9 :)